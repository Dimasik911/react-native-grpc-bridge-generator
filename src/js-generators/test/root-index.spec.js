/* eslint-env jest */

import generator from '../root-index'

describe('generator', () => {
  it('creates index', () => {
    const rsp = generator(['locations', 'booking', 'pricing'])

    expect(rsp).toMatchSnapshot()
    expect(rsp).not.toContain('undefined')
  })
})
