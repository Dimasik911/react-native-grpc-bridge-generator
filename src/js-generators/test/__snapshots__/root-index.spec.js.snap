// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generator creates index 1`] = `
"// @flow

const locations = require('./locations')
const booking = require('./booking')
const pricing = require('./pricing')

module.exports = {
  locations: locations,
  booking: booking,
  pricing: pricing
}"
`;
